resources/acl.rb

```diff
@@ -4,6 +4,8 @@
 property :config_dir, String, default: '/etc/haproxy'
 property :config_file, String, default: lazy { ::File.join(config_dir, 'haproxy.cfg') }
 
+unified_mode true
```

Comment:
does this break chef 14 builds?

Reply:
Everything but the very latest chef 14

Reply:
Ah, funnily enough I only tested it on the latest 14